airflow/providers/google/cloud/hooks/dataflow.py

```diff
@@ -1014,8 +1014,9 @@ def start_sql_job(
         self.log.info("Output: %s", proc.stdout.decode())
         self.log.warning("Stderr: %s", proc.stderr.decode())
         self.log.info("Exit code %d", proc.returncode)
+        stderr_last_20_lines = "\n".join(proc.stderr.decode().strip().splitlines()[-20:])
         if proc.returncode != 0:
-            raise AirflowException(f"Process exit with non-zero exit code. Exit code: {proc.returncode}")
+            raise AirflowException(f"Process exit with non-zero exit code. Exit code: \ {proc.returncode} Error Details : {stderr_last_20_lines}")
```

Comment:
`raise AirflowException(
                f"Process exit with non-zero exit code. Exit code: \ {proc.returncode} Error Details : {stderr_last_20_lines}"
            )`

Reply:
Thanks for your inputs @rohan472000 !